conn.go

```diff
@@ -84,7 +84,7 @@ func (c *conn) runQuery(ctx context.Context, query string) (driver.Rows, error)
 	// mode ctas
 	var ctasTable string
 	var afterDownload func() error
-	if isSelect && resultMode == ResultModeGzipDL {
+	if isCreateingCTASTable(isSelect, resultMode) {
```

Comment:
I guess it's a typo of `isCreatingCTASTable`, right? 😄

Reply:
Thanks! I fixed it.